packages/cli/src/html-to-jsx.ts

```diff
@@ -0,0 +1,168 @@
+    }
+  | { type: "element-end"; tagName: string };
+
+// eslint-disable-next-line func-style
+function* walkChildNodes(
+  node: DefaultTreeAdapterMap["node"]
+): IterableIterator<WalkNode> {
+  if (
+    defaultTreeAdapter.isCommentNode(node) ||
+    defaultTreeAdapter.isTextNode(node) ||
+    defaultTreeAdapter.isDocumentTypeNode(node)
+  ) {
+    throw new Error("Unsupported node type");
+  }
+
+  for (const childNode of node.childNodes) {
+    if (defaultTreeAdapter.isCommentNode(childNode)) {
+      continue;
+    }
+
+    if (defaultTreeAdapter.isTextNode(childNode)) {
+      yield { type: "text", value: childNode.value };
+      continue;
+    }
+
+    if (false === defaultTreeAdapter.isElementNode(childNode)) {
+      continue;
+    }
+
+    const attributes: [string, string][] = childNode.attrs.map((attr) => [
+      attr.name,
+      attr.value,
+    ]);
+
+    yield { type: "element-start", tagName: childNode.tagName, attributes };
+    yield* walkChildNodes(childNode);
+    yield { type: "element-end", tagName: childNode.tagName };
+  }
+}
+
+const convertStyleString = (style: string) => {
+  const styles = style
+    .split(";")
+    .map((style) => style.trim())
+    .map((style) => style.split(":").map((part) => part.trim()));
+
+  const res: Record<string, string> = {};
+  for (const [name, value] of styles) {
+    res[camelCase(name)] = value;
+  }
+  return JSON.stringify(res);
+};
+
+const toAttrString = (name: string, value: string) => {
+  const attName = name.toLowerCase();
+  const jsxName = attName === "class" ? "className" : attName;
+
+  if (value === "" && isBooleanAttr(attName)) {
+    return `${jsxName}`;
+  }
+
+  if (attName === "style") {
+    return `${jsxName}={${convertStyleString(value)}}`;
+  }
+
+  return `${jsxName}="${value}"`;
+};
+
+const attributesToString = (attributes: [string, string][]) =>
+  attributes
+    .map(([attName, value]) => ` ${toAttrString(attName, value)}`)
+    .join("");
+
+const convertTagName = (tagName: string) => {
+  const tag = tagName.toLowerCase();
+  if (tag === "script") {
+    return "Script";
+  }
+
+  if (tag === "style") {
+    return "Style";
+  }
+
+  return tag;
+};
+
+const escapeValue = (value: string) =>
+  value
+    .replace(/\\/g, "\\\\")
+    .replace(/`/g, "\\`")
+    .replace(/\$/g, "\\$")
+    .replace(/\r/g, "\\r")
+    .replace(/\n/g, "\\n");
+
+export const htmlToJsx = (html: string) => {
+  const parsedHtml = parseFragment(html, { scriptingEnabled: false });
+
+  let result = "";
+
+  for (const walkNode of walkChildNodes(parsedHtml)) {
+    switch (walkNode.type) {
+      case "text":
+        {
+          const escapedValue = escapeValue(walkNode.value);
+
+          result += escapedValue ? "{`" + escapedValue + "`}" : "";
+        }
+        break;
+      case "element-start":
+        {
+          const tag = convertTagName(walkNode.tagName);
+          result += `<${tag}${attributesToString(walkNode.attributes)}>`;
+        }
+        break;
```

Comment:
you can put break inside of block and it will format like if() {}

Reply:
yep done